rdkit/Chem/SaltRemover.py

```diff
@@ -74,15 +75,16 @@ def _initPatterns(self):
       inF = StringIO(self.defnData)
     else:
       inF = open(self.defnFilename, 'r')
-    self.salts = []
-    for line in inF:
-      line = line.strip().split('//')[0]
-      if line:
-        splitL = whitespace.split(line)
-        salt = Chem.MolFromSmarts(splitL[0])
-        if salt is None:
-          raise ValueError(line)
-        self.salts.append(salt)
+    with closing(inF):
+      self.salts = []
+      for line in inF:
+        line = line.strip().split('//')[0]
```

Comment:
yeah, the split method always returns something, so taking the 0th element is always ok.